ratpack-session/src/test/groovy/ratpack/session/clientside/ClientSideSessionSpec.groovy

```diff
@@ -337,62 +343,14 @@ class ClientSideSessionSpec extends SessionSpec {
 
     expect:
     text == "null"
-    postText("set/foo") == "ok"
+    result.set(postText("set/foo"))
```

Comment:
This shouldn't be necessary. postText() is already synchronous.

Reply:
Removed!